beater/config.go

```diff
@@ -15,6 +15,7 @@ type Config struct {
 	SSL                *SSLConfig    `config:"ssl"`
 	ConcurrentRequests int           `config:"concurrent_requests" validate:"min=1"`
 	EnableFrontend     bool          `config:"enable_frontend"`
+	Origin             string        `config:"request_origin"`
```

Comment:
YAML has a way to specify lists. We could probably utilize that to make it simple to add multiple here?

I also think we should name the configuration option something like `AllowOrigin`.

Reply:
Sorry, i realize that is (4) from above. Seems like it would be simple to add, so I'm +1 on that.